hv-rhel6.x/hv/Makefile

```diff
@@ -37,6 +37,9 @@ ccflags-y += -I$(M)/include/ -I$(M)/arch/$(SRCARCH)/include -I$(M)/arch/$(SRCARC
 # Get extra kernel version information beyond LINUX_VERSION_CODE or RHEL_RELEASE_CODE.
 # This is used to identify RHEL/CentOS kernel update versions.
 extraversioninfo = $(shell uname -r | awk -F'-' '{print $$2}')
+ifdef KERNEL_VERSION
+	extraversioninfo = $(KERNEL_VERSION)
```

Comment:
If KERNEL_VERSION is defined, we should also pipe $(KERNEL_VERSION) into the awk to find the additional version info.